api/controllers/userController.js

```diff
@@ -0,0 +1,10 @@
+const { userService } = require('../services')
+const { catchAsync } = require('../utils/error')
+
+const kakaoSignIn = catchAsync(async(req, res) => {
+    const code = req.headers.authorization
```

Comment:
인가 코드로 우리 서비스 서버의 자격 증명을 확인하는 것이 아니기 때문에
query 혹은 code 자체를 받아도 될 것 같습니다.

```suggestion
    const code = req.query
```